examples/index.html

```diff
@@ -44,6 +45,10 @@
             loader.add('pug', 'pug.jpg')
                 .load(function(loader, resources){
                     var texture = resources.pug.texture;
+                    sprite.scale.x = 0.9;
+                    sprite.scale.y = 0.9;
+                    sprite.x = 5;
+                    sprite.y = 5;
```

Comment:
Rather than scaling down the sprite, we should just make the canvas a little larger than the image.